File: modul-04/bahasa c/PRAK403-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main()
{
    int nilai1, nilai2;
    int nilai_batas;

    scanf("%d %d", &nilai1, &nilai2);
    nilai_batas = (nilai1 - nilai2);
    if (nilai_batas < 0)
    {
        nilai_batas = nilai_batas * -1;
        for (int b = 0; b <= nilai_batas; b++)
        {
            int a = 1;

            printf("%d-%d ", nilai1, nilai2);

            nilai1 = nilai1 + a;
            nilai2 = nilai2 - a;
        }
    }
    else
    {
        for (int b = 0; b <= nilai_batas; b++)
        {
            int a = 1;

            printf("%d %d  ", nilai1, nilai2);

            nilai1 = nilai1 - a;
            nilai2 = nilai2 + a;
        }
    }

    return 0;
}

File: modul-04/python/PRAK403-2310817310003-Mfaridsetiawan.py
nilai1, nilai2 = map(int, input().split())
nilai_batas = nilai1 - nilai2

if nilai_batas < 0:
    nilai_batas = abs(nilai_batas)
    for i in range(nilai_batas + 1):
        a = 1

        print(f"{nilai1}-{nilai2} ", end="")

        nilai1 += a
        nilai2 -= a
else:
    for i in range(nilai_batas + 1):
        a = 1

        print(f"{nilai1} {nilai2}  ", end="")

        nilai1 -= a
        nilai2 += a